compute/client_library/ingredients/instances/ip_address/promote_ephemeral_ip.py

```diff
@@ -0,0 +1,49 @@
+#  Copyright 2024 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# flake8: noqa
+import uuid
+
+from google.cloud.compute_v1 import AddressesClient
+from google.cloud.compute_v1.types import Address
+
+# <INGREDIENT promote_ephemeral_ip>
+
+
+def promote_ephemeral_ip(project_id: str, ephemeral_ip: str, region):
+    """
+    Promote the first ephemeral IP found on the instance to a static IP.
```

Comment:
first? I think we're pointing to a very specific IP by its number. There won't be 2 ephemeral IPs with the same value, right?

Reply:
Corrected